cmd/statusd/utils.go

```diff
@@ -19,13 +19,13 @@ import (
 
 	"fmt"
 
-	"github.com/status-im/status-go/geth/account"
+	"github.com/status-im/status-go/geth/accounts"
```

Comment:
Why this `account`->`accounts` rename and why in this PR? Is it related to the test refactor?

Actually, I was thinking about this plural option, and I think the common approach is to use singular form - `image`, `log`, `node`, etc.

Reply:
I am not sure why I did it. I think it was to align with `go-ethereum`. I can revert it to `geth/account`.

Reply:
Yeah, let's revert for now. Actually, maybe `accounts` is better, but anyway it should be discussed and done in separate PR. The pain of merging this change with other open branches will be huge. :)

Reply:
Ok, I will revert. True, that's why I'd like to merge geth 1.7.0 and your refactoring first.